src/content/docs/tutorial-service-level-mgmt/establish-output-sli.mdx

```diff
@@ -0,0 +1,100 @@
+
+<img
+  alt="Service map example"
+  src={apmServiceBoundary}
+/>        
+
+<Callout variant="tip">
+Your team may only be accountable for the **Order-Composer** dependency. If so, then your own service level on **Order-Composer** can self-monitor the performance. Ensure you tag your own non-customer facing service levels as `customer-facing:false` to allow for better filtering in health reports. Also, consider working with the customer-facing endpoint (#1 Order-Processing) during observability to create true output performance, an input connectivity service level, and client service levels.
+</Callout>
+
+    </Step>
+    <Step>
+## Create your baseline
+
+You must also establish a baseline to implementat service levels. Creating a baseline allows you to measure the current performance of a service, then use service level reports to find out if you're hitting that baseline or degrading. You can create a baseline for almost any dataset; however, you use different formulas and recommendations in different use cases.
+
+When you create service levels, you should start with the output performance of your applications. You can use response times (latency) and percentage of non-errors (success). When you do this, you create a reliability health metric. Seasonality and peak usage doesn't handicap your performance, and the more history you include in your measurement, the more likely you're including different codebases from releases. Previous deployments, no matter how small, could skew your results.
+
+We recommend a history of one to two weeks of performance data to establish a good baseline. Here's an example NRQL query for the recommended target of a seven day service level objective for latency:
+
+```
+FROM Transaction SELECT percentile(duration, 95) AS 'Latency Baseline SLI' WHERE appName='Order-Processing' SINCE 1 WEEK AGO
+```
+
+<img
+  alt="Latency Baseline"
+  src={apmLatencyBaseline}
+/>
+
+For a successful (error-free) baseline, try the following query. Remember to substitute `Order-Processing` for your own application name.
+
+```
+FROM Transaction SELECT percentage(count(*), WHERE error is false) AS 'Success Baseline SLI' SINCE 1 WEEK AGO WHERE appName='Order-Processing'
+```        
+    </Step>
+    <Step>
+## Create your service level
+
+**Note:** If you don't see the **Add a service level** button, check with your New Relic administrator about your permissions.
+
+Our platform automatically calculates the recommended <InlinePopover type="apm" /> and browser baselines for you. The identify your service section above shows you how to find your application APM data. You'll see #2 in the screenshot in that same section, called "Service levels." Find your application APM data and click **Service levels**, and you should see the view below:
```

Comment:
Should "identify your service section" be bolded? i'm not sure what this is referring to